src/common/domain/openInZeplin/util/zeplinUrls.ts

```diff
@@ -106,12 +106,74 @@ function getComponentSectionAppUrl(barrelId: string, barrelType: BarrelType, com
     return `${configuration.appUrl}components?cseid=${componentSectionId}&${barrelIdKey}=${barrelId}`;
 }
 
+/**
+ * Returns a screen's app url
+ * @param projectId A project id.
+ * @param screenId A screen id.
+ * @param applicationType Type of app to create url of.
+ */
+function getScreenUrl(projectId: string, screenId: string, applicationType: ApplicationType): string {
+    return applicationType === ApplicationType.Web
+        ? getScreenWebUrl(projectId, screenId)
+        : getScreenAppUrl(projectId, screenId);
+}
+
+/**
+ * Returns a screen's Web app url
+ * @param projectId A project id.
+ * @param screenId A screen id.
+ */
+function getScreenWebUrl(projectId: string, screenId: string): string {
+    return `${getProjectWebUrl(projectId)}/screen/${screenId}`;
+}
+
+/**
+ * Returns a screen's Mac app url
```

Comment:
```suggestion
 * Returns a screen's Mac app uri
```

Reply:
Selim also mentioned this in another PR, I will make this change in a few places, so I will open a separate PR for this